### PR TITLE
[Ascend] fuj/aclnn-replace

### PR DESCRIPTION
**Pull request by jingguo-st**
## Motivation and Context

replace op_plugin with aclnn

## Description

1. diopiMul
2. diopiAdd
3. diopiSub
4. diopiPow
5. diopiDiv
6. diopiNorm...

## Checklist
**Before PR**:

- [x] I have read and followed the workflow indicated in the [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) to create this PR.
- [x] Pre-commit or linting tools indicated in [Contributors.md](https://github.com/DeepLink-org/DIOPI/blob/main/Contributors.md) are used to fix the potential lint issues.
- [ ] Bug fixes are covered by unit tests, the case that causes the bug should be added in the unit tests.
- [ ] New functionalities are covered by complete unit tests. If not, please add more unit test to ensure the correctness.
- [ ] The documentation has been modified accordingly, including docstring or example tutorials.

**After PR**:

- [ ] CLA has been signed and all committers have signed the CLA in this PR.

### CODE REVIEW

File: impl/ascend_npu/CMakeLists.txt [hunk 1]
```diff
@@ -181,6 +181,7 @@ set(OLD_IMPL_SRC
     ${OLD_IMPL_DIR}/functions/reduce.cpp
     ${OLD_IMPL_DIR}/functions/where.cpp
     ${OLD_IMPL_DIR}/functions/multinomial.cpp
+    ${OLD_IMPL_DIR}/functions/mul.cpp
```
Comment (yewentao256) [inline impl/ascend_npu/CMakeLists.txt:184]: 这里mul.cpp加在ascend文件夹下，但却在ascend_npu的cmakelist更新，蛮奇怪的

Comment (jingguo-st) [reply impl/ascend_npu/CMakeLists.txt:185]: 后面去除op_plugin的时候再整合吧， 现在ascend的编译是通过ascend_npu下的CMakeLists配置的

**Review by zhangzefeng92 — APPROVED** [review]

**Review by NeosZhang — APPROVED** [review]

File: impl/ascend/functions/normal.cpp [hunk 2]
```diff
@@ -4,92 +4,79 @@
...
-    diopiAddInpScalar(ctx, out, &meanScalar, &alphaScalar);
+    float meanCast = static_cast<float>(mean);
+    float rstdCast = static_cast<float>(std);
+    DIOPI_ASCEND_CALL_ACLNN(aclnnNormalFloatFloat, ctx, meanCast, rstdCast, seed, offset, out);
     return diopiSuccess;
 }
 
 diopiError_t diopiNormalInp(diopiContextHandle_t ctx, diopiTensorHandle_t inout, double mean, double std, diopiGeneratorHandle_t generator) {
-    return diopiNormal(ctx, inout, mean, std, generator);
+    auto pair = getSeedAndOffset(ctx, generator, 10);
```
Comment (yangbofun) [inline impl/ascend/functions/normal.cpp:29]: 改为
    uint64_t seed, offset;
    DIOPI_CALL(diopiGeneratorGetSeedAndOffset(generator, seed, offset));
下同。

Comment (jingguo-st) [reply impl/ascend/functions/normal.cpp]: fix

File: impl/ascend/functions/pow.cpp [hunk 3]
```diff
@@ -1,35 +1,37 @@
 /**
  * @file
  * @author DeepLink
- * @copyright  (c) 2023, DeepLink.
+ * @copyright  (c) 2024, DeepLink.
```
Comment (yangbofun) [inline impl/ascend/functions/pow.cpp:4]: 这个不用改

**Pull request merged**